### PR TITLE
Never create merge_commit_* tags

### PR DESCRIPTION
**Pull request by foolip**
This has happened in production, creating a number of these tags.

The tags have already been deleted.

### CODE REVIEW

**Comment by foolip** [comment]
@jgraham can you review?

I've deleted the misnamed tags and releases once, but one of them came back. It's the one that https://github.com/web-platform-tests/wpt/runs/127493402 created, maybe someone had the tag locally and pushed it.

**Review by jgraham — APPROVED** [review]
OK, but I don't really understand what problem this is causing. I'm more worried by the fact that the commit you mentioned seems to be tagged with the wrong PR?

**Comment by foolip** [comment]
> OK, but I don't really understand what problem this is causing.

They're just adding noise when trying to audit if we've successfully tagged and uploaded assets since the switch to GitHub Actions. We won't try to use anything attached to a release for a tag named like this because of this filtering:
https://github.com/web-platform-tests/wpt/blob/e98eee5aec6ba154f2f086e9d2f0a287617e8519/tools/manifest/download.py#L60

> I'm more worried by the fact that the commit you mentioned seems to be tagged with the wrong PR?

Do you mean in https://github.com/web-platform-tests/wpt/runs/127493402, commit 860b5942703a6aa1dd9dde8d049a55787dd88c21? I don't know for sure what went wrong there, but given the types of bugs we've seen before, I would guess that merged PR hadn't propagated to the GitHub search API yet.

Holding off merging to make sure I understand the question.

**Comment by jgraham** [comment]
I thinj I was just misunderstanding the GH UI. Please go ahead and merge.

**Pull request merged**
